psalm-baseline.xml

```diff
@@ -176,6 +179,9 @@
     </PropertyNotSetInConstructor>
   </file>
   <file src="test/ConfigProviderTest.php">
+    <DeprecatedClass>
+      <code>new Config($dependencies)</code>
+    </DeprecatedClass>
```

Comment:
This deprecation is pretty easy to deal with, with SM at ^3, we can just pass a regular array to the SM constructor and remove the usage of the `Config` class in the test. Could you take a look at that whilst you're here please?

See https://github.com/mezzio/mezzio/blob/2bac6f637837fce15239a358b7e13a71c400d5c4/test/ConfigProviderTest.php#L139-L146

Pretty sure this method can just be `return new ServiceManager($dependencies)`

Also, the `psalm-type` is outdated here - it should probably be `@psalm-import-type ServiceManagerConfiguration from ServiceManager`

Reply:
done